src/sentry/notifications/utils/__init__.py

```diff
@@ -129,6 +130,30 @@ def summarize_issues(
     return rv
 
 
+def get_email_link_extra_params(
```

Comment:
Let's add a test for this

Reply:
Also wondering if this can be shared with `get_group_settings_link`, since they're doing similar things.

Reply:
Not sure if this is the right place for it but:
https://github.com/getsentry/sentry/blob/565903080c45097a321c028a027dc882fb4c046c/tests/sentry/notifications/test_utils.py#L167